app/src/main/java/net/gsantner/opoc/util/StringUtils.java

```diff
@@ -298,4 +305,22 @@ public static int tryParseInt(final String value, int defaultValue) {
             return defaultValue;
         }
     }
+
+    // So one doesn't have to define all three always
+    public static class SimpleTextWatcher implements TextWatcher {
```

Comment:
Recommendation: TextWatcherAdapter (i.e. normal java interface listener vs adapter)

Reply:
Just got rid of it. Only used in 2 places anyway.